packages/plaid-funding-source/pages/index.tsx

```diff
@@ -32,6 +32,26 @@ const Home: NextPage = () => {
      */
     const [linkToken, setLinkToken] = useState<string | null>(null);
 
+    /**
+     * The body text that is returned from the /on-demand-authorizations endpoint
+     */
+    const [bodyText, setBodyText] = useState<string | null>(null);
+
+    /**
+     * The button text that is returned from the /on-demand-authorizations endpoint
+     */
+    const [buttonText, setButtonText] = useState<string | null>(null);
+
+    /**
+     * The ODA link that is returned from the /on-demand-authorizations endpoint
+     */
+    const [odaLink, setOdaLink, odaLinkRef] = useState<string | null>(null);
+
+    /**
+     * Controls checkbox for on demand authorization
+     */
+    const [isChecked, setIsChecked] = useState<boolean | false>(false);
```

Comment:
Can we use `setChecked` instead of `setIsChecked`? `is` should just prefix the accessor. 

Also, `<boolean | false>` can be reduced to `<boolean>` since that will include `false` as a value.

Reply:
We can. I usually stick with the convention of prefixing boolean values with "is". Since the project is using TypeScript, I can end up changing the values to [checked, setChecked] instead as the type is explicit.